pkgs/tools/misc/kronometer/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  kdeDerivation, kdeWrapper, fetchurl, lib,
+  ecm, kdoctools,
+  kconfig, kinit
+}:
+
+let
+  unwrapped = let
+    pname = "kronometer";
+    version = "2.1.0";
+  in kdeDerivation rec {
+    name = "${pname}-${version}";
+
+    src = fetchurl {
+      url = "mirror://kde/stable/${pname}/${version}/src/${name}.tar.xz";
+      sha256 = "1nh7y4c13rscy55f5n8s2v8jij27b55rwkxh9g8r0p7mdwmw8vri";
+    };
+
+    meta = {
+      license = with lib.licenses; [ gpl2 lgpl21 fdl12 ];
```

Comment:
Are you sure? I've not seen that before, not that it would break anything since it's just metadata.